modules/kameleonvolume/kameleonvolumereader.cpp

```diff
@@ -181,15 +230,76 @@ ghoul::Dictionary KameleonVolumeReader::readMetaData() const {
     };
 }
 
-float KameleonVolumeReader::minValue(const std::string & variable) const {
-    return _model->getVariableAttribute(variable, "actual_min").getAttributeFloat();
+std::string KameleonVolumeReader::simulationStart() const {
+    std::string startTime;
+    if (_model->doesAttributeExist("start_time")){
+        startTime =
+            _model->getGlobalAttribute("start_time").getAttributeString();
+    } else if (_model->doesAttributeExist("tim_rundate_cal")) {
+        startTime =
+            _model->getGlobalAttribute("tim_rundate_cal").getAttributeString();
+        size_t numChars = startTime.length();
+        if (numChars < 19) {
+            // Fall through to add the required characters
+            switch (numChars) {
+                case 10 : // YYYY-MM-DD        => YYYY-MM-DDTHH
+                    startTime += "T00";
```

Comment:
Since we switched to C++17 now, you can use `[[fallthrough]]` on the last line of the case label to make it clear to the compiler that a fallthrough is intentional.  Otherwise, it will start warning about it at some point